style.css

```diff
@@ -118,4 +118,423 @@ div.contact ul a {
+  font-size: 18px;
+  margin-left: 6.5%;
+  margin-bottom: 50px;
+}
+
+button.see:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+button.see:active {
+  background: #e05304;
+  color: #fff5e1;
+}
+
+button.see:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+/* profitional part */
+div.subjects {
+  position: relative;
+  background: #fff;
+}
+
+div.profesional {
+  margin: 0%;
+  padding: 0%;
+  margin-left: 3%;
+  width: 330px;
+  height: 386px;
+  position: absolute;
+  background-image: url(images/Img\ Placeholder1.png);
+}
+
+div.profesional h3 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 26px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #f1f3f7;
+  margin-left: 6.5%;
+  margin-top: 80px;
+}
+
+.profesional p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 28px;
+  color: #f1f4f8;
+  margin-left: 6.5%;
+  margin-top: -15px;
+}
+
+div.language {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-left: 2%;
+  margin-bottom: 2%;
+}
+
+div.language span {
+  box-sizing: border-box;
+  padding: 10px 12px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-right: 5%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+button.see-project {
+  border: none;
+  outline: none;
+  color: #fff;
+  background: #ff6b00;
+  padding: 10px;
+  font-size: 18px;
+  width: 330px;
+  margin-left: 0%;
+}
+
+button.see-project:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+button.see-project:active {
+  background: #e05304;
+  color: #fff5e1;
+}
+
+button.see-project:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+div.p-2 {
+  margin-top: 444px;
+}
+
+div.p-3 {
+  margin-top: 888px;
+}
+
+div.p-4 {
+  margin-top: 1332px;
+}
+
+div.p-5 {
+  margin-top: 1776px;
+}
+
+div.p-6 {
+  margin-top: 2220px;
+  margin-bottom: 100px;
+}
+
+/* section works end */
+
+/* section about start */
+.about {
+  position: absolute;
+  width: 378px;
+  height: 1801px;
+  left: 0;
+  top: 4500px;
+  background: #1c1a19;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+section.about h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 80px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #ff6b00;
+  margin-top: 40%;
+}
+
+.about p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 28px;
+  color: #f1f4f8;
+  margin-left: -4%;
+  margin-top: -30px;
+  width: 320px;
+}
+
+button.get {
+  border: none;
+  outline: none;
+  color: #fff;
+  background: #ff6b00;
+  padding: 10px;
+  font-size: 18px;
+  margin-left: -47.5%;
```

Comment:
- [ ] Great job with your project and styling 👏 . As the previous reviewer mentioned kindly add the `button interactions` to the `Get my Resume` `button` 🙏 

![image](https://user-images.githubusercontent.com/74814780/193081918-0fe68f5b-5f39-4f09-89ad-a2f51320bf64.png)